tests/integration-tests/tests/iam/test_iam_image.py

```diff
@@ -88,6 +87,20 @@ def _wait_build_image_complete(image):
     assert_that(image.image_status).is_equal_to("BUILD_COMPLETE")
 
 
+@retry(wait_fixed=minutes(1), stop_max_delay=minutes(60))
```

Comment:
I'd reduce the `stop_max_delay` to 20-25 minutes, since 10 was enough most of the times.

Reply:
I can do that.  It usually succeeds on the second or third retry in my local testing